user_management/src/user/migrations/0002_user_email_user_password.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.7 on 2023-12-14 18:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('user', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='email',
+            field=models.EmailField(default='default@default.fr', max_length=254),
```

Comment:
Make the max email length the same here and in user_management/src/user/views.py::is_valid_email()

Reply:
Was not resolved in latest commit

Reply:
Add `email = models.EmailField(max_length={settings.MAX_EMAIL_LENGTH})` in user_management/src/user/models.py